nomad/serf_test.go

```diff
@@ -327,16 +326,35 @@ func TestNomad_BootstrapExpect_NonVoter(t *testing.T) {
 	})
 	defer cleanupS4()
 
+	servers := []*Server{s1, s2, s3, s4}
+
 	// Join with fourth server (now have quorum)
 	// Start with 4th server for higher chance of success
```

Comment:
This line of the comment is no longer relevant I think?

```suggestion
```

Reply:
Oh I didn't notice that the order was important. I pushed a commit to store the server in reverse order and moved this comment 👍